docs/volto-light-theme/index.md

```diff
@@ -0,0 +1,32 @@
+---
+myst:
+  html_meta:
+    "description": "Volto Light Theme"
+    "property=og:description": "Volto Light Theme"
+    "property=og:title": "Volto Light Theme"
+    "keywords": "Plone, Volto, Training"
+---
+
+(volto-light-theme-label)=
+
+# Volto Light Theme
+
+About
+: Learn how to properly use the volto-light-theme in your project.
+
+Level
+: Beginner level
```

Comment:
I'm not sure what you are trying to say here. Why is this a definition list, instead of friendly narrative text that speaks directly to the participant?